src/java/main/org/apache/zookeeper/server/util/ConfigUtils.java

```diff
@@ -61,10 +61,16 @@ static public String getClientConfigStr(String configData) {
         return version + " " + sb.toString();
     }
 
+    /**
+     * Splits server config to server and port
+     * with support for IPv6 literals
+     * @return String[] first element being the
+     *  IP address and the next being the port
+     * @param s server config, server:port
+     */
     public static String[] splitServerConfig(String s)
         throws ConfigException
```

Comment:
`splitServerConfig` is somewhat general. `getHostAndPort` is better?

Reply:
Sure, I like that better too - I've renamed this